MAUI/Maps/migration.md

```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
+                         <map:MapPolyline StrokeThickness="3" Stroke="Blue">
+                             <map:MapPolyline.Points>
+                                 <map:MapLatLng Latitude="80.2707"
+                                                Longitude="13.0827" />
+                                 <map:MapLatLng Latitude="79.6117"
+                                                Longitude="13.1746" />
+                                 <map:MapLatLng Latitude="79.5037"
+                                                Longitude="13.6373" />
+                                 <map:MapLatLng Latitude="78.8242"
+                                                Longitude="14.4673" />
+                             </map:MapPolyline.Points>
+                         </map:MapPolyline>
+                     </map:MapPolylineLayer.Polylines>
+                 </map:MapPolylineLayer>
+             </map:MapShapeLayer.Sublayers>
+         </map:MapShapeLayer>
+     </map:SfMaps.Layer>
+ </map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+ public MainPage()
+ {
+     InitializeComponent();
+     SfMaps maps = new SfMaps();
+     MapShapeLayer layer = new MapShapeLayer();
+     layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/india.json"));
+     MapPolylineLayer mapPolylineLayer = new MapPolylineLayer();
+     MapPolyline polyline = new MapPolyline();
+     polyline.Points = new ObservableCollection<MapLatLng>()
+      {
+          new MapLatLng(80.2707, 13.0827),
+          new MapLatLng(79.6117, 13.1746),
+          new MapLatLng(79.5037, 13.6373),
+          new MapLatLng(78.8242, 14.4673),
+      };
+     polyline.Stroke = Colors.Blue;
+     mapPolylineLayer.Polylines.Add(polyline);
+     layer.Sublayers.Add(mapPolylineLayer);
+     maps.Layer = layer;
+     this.Content = maps;
+ }
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+### Line Layer
+
+<table>
+<tr>
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'MapLine'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'From'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'To'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Stroke'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+     {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeThickness'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeDashArray'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeLineCap'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```

Comment:
The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

Reply:
Modified